API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -32,6 +32,53 @@ GeoJSON does not cover all use cases. For example, it is not possible to store c
 - When supporting GML, do this according to OGC API Features [Requirements class 8.4](https://docs.ogc.org/is/17-069r3/17-069r3.html#_requirements_class_geography_markup_language_gml_simple_features_profile_level_0) for GML Simple Features level 0, or [Requirements class 8.4](https://docs.ogc.org/is/17-069r3/17-069r3.html#_requirements_class_geography_markup_language_gml_simple_features_profile_level_2) for GML Simple Features level 2. 
 - Use a workaround, e.g. convert circular lines / arcs to regular linestrings. 
 
+Example of embedding WKT in a JSON object:
+<pre>
+// definition of JSON object
```

Comment:
Regels 38 t/m 42 kunnen weg. De // aan het begin van de regels mogen ook weg. Regels 43 t/m 50 kunnen 2 spaties naar links.

Reply:
Dit is aangepast. Ik heb geprobeerd het wat duidelijker te maken door de definitie van het JSON object naar een eigen \<pre\> blok te verplaatsen